framework/console/widgets/Table.php

```diff
@@ -376,6 +377,9 @@ protected function getScreenWidth()
             if (isset($size[0])) {
                 $this->_screenWidth = $size[0];
             }
+            else {
+                $this->_screenWidth = 123;
```

Comment:
Why exactly 123?

Reply:
https://github.com/yiisoft/yii2/blob/0948109a19a59c63d4f065343fe530421764d768/framework/console/widgets/Table.php#L301
At line 301 it decreases by 3 (i don't know why. For terminal scrollbar?) and became 120. I think 120 is "standard" line length. May be, it could be larger for log files.

Reply:
OK.